### PR TITLE
BZ-1949009: Moving port requirements for control plane machines only

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://bugzilla.redhat.com/show_bug.cgi?id=1949009

Preview: https://deploy-preview-32039--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-network-customizations.html#installation-network-user-infra_installing-vsphere-network-customizations

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@hexfusion @geliu2016 Can you please review?

Preview: https://deploy-preview-32039--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-network-customizations.html#installation-network-user-infra_installing-vsphere-network-customizations

File: modules/installation-network-user-infra.adoc [hunk 1]
```diff
@@ -131,15 +131,26 @@ the Cluster Version Operator on port `9099`.
...
+[cols="2a,2a,5a",options="header"]
+|===
+
+|Protocol
+|Port
+|Description
+
+|TCP
+|`2379`-`2380`
+|etcd server, peer, and metrics ports
```
Comment (hexfusion) [inline modules/installation-network-user-infra.adoc:150]: please drop metrics that is now `9979` covered in separate section.

Comment (hexfusion) [reply modules/installation-network-user-infra.adoc:150]: otherwise LGTM

Comment (bergerhoffer) [reply modules/installation-network-user-infra.adoc]: Done, thanks

**Comment by hexfusion** [comment]
LGTM

**Comment by hexfusion** [comment]
probably need to backport this as far back as is supported.

**Comment by bergerhoffer** [comment]
> probably need to backport this as far back as is supported.

@hexfusion Yes I have this marked to go back as far as possible, which is 4.5.

**Comment by geliu2016** [comment]
LGTM,thanks

**Pull request merged**